config/sql/schema.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE tls_creds(ca_cert BLOB, ca_cert_format TEXT,
                        client_cert BLOB, client_cert_format TEXT,
                        client_pkey BLOB, client_pkey_format TEXT);
 CREATE TABLE meta(meta BLOB NOT NULL, repo INTEGER NOT NULL, meta_type INTEGER NOT NULL, version INTEGER NOT NULL, UNIQUE(repo, meta_type, version));
-CREATE TABLE target_images(filename TEXT UNIQUE, image_data BLOB NOT NULL);
+CREATE TABLE target_images(filename TEXT UNIQUE, image_data BLOB NOT NULL, real_size INTEGER NOT NULL DEFAULT 0);
```

Comment:
If you change the SQL schema, you will also need to write a migration script. We've also decided that we need to be able to migrate in reverse as well. We don't need to be able to go back further than where we are now (because it's too late for that to be meaningful), but from this point on, we have to be able to migrate forwards and backwards.

Reply:
Yes I know and I did it. I just didn't include it in the commit